### ISSUE TITLE
Run Accessibility tests on new Data.gov home page

### ISSUE THREAD

**Issue by hkdctol** [GSA/data.gov#4250]
## User Story

In order to meet GSA requirements for site launch, Data.gov team needs to run accessibility tests on the new version of the home page.

## Acceptance Criteria

- [ ] GIVEN the redesigned home page on Cloud.gov Pages \
  THEN the site has been tested for Accessibility requirements\
  AND any findings have been remediated

## Sketch

- [x] Use GSA AMP tool to run tests on new home page
- [x] Review findings
- [x] Remediate findings

**Comment by nickumia-reisys** [comment]
@btylerburton ran the AMP Report yesterday, the results are in the [google drive](https://drive.google.com/file/d/1j5x9xuFY15W-zWvLLSNkCEa1ax2I31oI/view).

There was just one non-compliant item which was the duplication of `svg` resources.  I will tackle that today.

**Comment by btylerburton** [comment]
All high level violations have been mitigated. New report from 3.23 stored in [Accessibility folder](https://drive.google.com/drive/folders/1THQWMi1tVFh_2nj2h2rDchwa2KlF3glK) in Google Drive.
